services/yolov3-object-detection/service/service_spec/object_detection.proto

```diff
@@ -6,12 +6,19 @@ message Input {
     string confidence = 3;
 }
 
+message Boxes {
+    float x = 1;
+    float y = 2;
+    int32 w = 3;
```

Comment:
Minor, but seems a bit strange that `x` and `w` or `y` and `h` has different types.

Reply:
`x` and `y` come from a division by 2 ([here](https://github.com/singnet/dnn-model-services/blob/master/services/yolov3-object-detection/service/object_detection.py#L97)) but I agree with you.
I'll `round()` them and change the `.proto` to `int32`.
Thanks!